### PR TITLE
add component a11y test blueprints for docs and components

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

adds a component a11y test blueprints for the website and the components package that will generate these tests when running existing generators

### :link: External links

Jira ticket: [HDS-2529](https://hashicorp.atlassian.net/browse/HDS-2529)

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-2529]: https://hashicorp.atlassian.net/browse/HDS-2529?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/blueprints/hds-component-docs/files/tests/acceptance/components/__name__.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+module('Acceptance | components/<%= dasherizedModuleName %>', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/<%= dasherizedModuleName %>', async function (assert) {
+    await visit('/components/<%= dasherizedModuleName %>');
+
+    assert.strictEqual(currentURL(), '/components/<%= dasherizedModuleName %>');
+  });
+
+  test('Components/<%= dasherizedModuleName %> page passes a11y automated checks', async function (assert) {
```
Comment (MelSumner) [inline website/blueprints/hds-component-docs/files/tests/acceptance/components/__name__.js:20]:
```suggestion
  test('Components/<%= dasherizedModuleName %> page passes automated a11y checks', async function (assert) {
```

**Review by MelSumner — CHANGES_REQUESTED** [review]
Just requesting one tiny change based on a change I implemented from a code review.

File: website/blueprints/hds-component-docs/files/tests/acceptance/components/__name__.js (hunk 1 above)
Comment (Dhaulagiri) [reply website/blueprints/hds-component-docs/files/tests/acceptance/components/__name__.js:20]: I think we have an inconsistency between how these are worded in the components project vs the website project. Happy to make this change, but future website tests will not match the previous website tests.

**Comment by Dhaulagiri** [comment]
@MelSumner would it makes sense to also create a blueprint for adding this type of test to _component_ acceptance tests?

**Comment by MelSumner** [comment]
@Dhaulagiri yes, since we're adding them now. 👍

**Review by MelSumner — APPROVED** [review]
:shipit:

**Comment by Dhaulagiri** [comment]
Since I was already doing it and am not good reading comprehension I made the changes in this PR (but can pull it out if it would be helpful).

I personally would not leave in the tests that assert on the URL, but I did so to match the existing convention.

**Review by MelSumner — APPROVED** [review]

**Pull request merged**